app/models/participant_id_change.rb

```diff
@@ -2,8 +2,8 @@
 
 class ParticipantIdChange < ApplicationRecord
   belongs_to :user
-  belongs_to :from_participant, class_name: "User"
-  belongs_to :to_participant, class_name: "User"
+  belongs_to :from_participant, class_name: "User", primary_key: "ecf_id", optional: true
+  belongs_to :to_participant, class_name: "User", primary_key: "ecf_id", optional: true
```

Comment:
I don't think we should have `belongs_to` for `from_participant` and `to_participant` as we shouldn't ever use them to find users.

Instead add validation presence for `from_participant_id` and `to_participant_id`

Reply:
that's fair actually 👍